### PR TITLE
minor graphcast fixes

### PR DESCRIPTION
**Pull request by mnabian**
# Modulus Pull Request

Fixing minor issues in GraphCast training

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [ ] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [ ] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

**Review by ktangsali — APPROVED** [review]

File: examples/weather/graphcast/train_graphcast.py [hunk 1]
```diff
@@ -45,15 +45,16 @@
...
-
 # Instantiate constants, and save to JSON file
 C = Constants()
 
+# Optionally import apex
+if C.use_apex:
+    try:
+        import apex
+    except:
+        pass
```
Comment (Alexey-Kamenev) [inline examples/weather/graphcast/train_graphcast.py:56]: So now that there is a dedicated `use_apex` option, would not user want to know that apex will not be used despite this option being "on"? Maybe add a warning message or don't silence the exception?

File: examples/weather/graphcast/train_graphcast.py [hunk 2]
```diff
@@ -235,7 +237,8 @@ def __init__(self, wb, dist, rank_zero_logger):
     )  # Wandb logger
     logger = PythonLogger("main")  # General python logger
     rank_zero_logger = RankZeroLoggingWrapper(logger, dist)  # Rank 0 logger
-    logger.file_logging()
+    if dist.rank == 0:
```
Comment (Alexey-Kamenev) [inline examples/weather/graphcast/train_graphcast.py:240]: Just curious, can you simply use this:
```py
 rank_zero_logger.file_logging()
```
?

**Review by Alexey-Kamenev — APPROVED** [review]

File: examples/weather/graphcast/train_graphcast.py (hunk 1 above)
Comment (mnabian) [reply examples/weather/graphcast/train_graphcast.py]: Thanks, added an import error.

**Pull request merged**